src/components/Header/Header.js

```diff
@@ -1,12 +1,54 @@
-import React from "react";
+import { React, useState } from "react";
+
+import history from '../../utils/history';
+import routePaths from '../../constants/routePaths';
+
 
 const Header = () => {
     
+  const [dropdown, setDropdown] = useState(false);
+
+  const toggleDropdown = (bool) => {
+    setDropdown(!bool);
+  };
+
+  const dropdownMenu = (
```

Comment:
Can we write this in a cleaner way? This has a lot of repetition

Reply:
I'm spending too long on this part - okay if I get back to this and the bootstrap issue in another PR?